lib/Doctrine/Common/Proxy/AbstractProxyFactory.php

```diff
@@ -47,11 +48,22 @@ abstract class AbstractProxyFactory
      */
     public const AUTOGENERATE_EVAL = 3;
 
+    /**
+     * Autogenerate the proxy class when the proxy file does not exist or
+     * when the entity changed.
```

Comment:
@alcaeus @franmomu is "entity" the correct word here? Or is this class used for the ODM as well? There are 2 other occurrences of "entity" in that project.

Reply:
Fixed